.azure/pipelines/signalr-daily-tests.yml

```diff
@@ -4,6 +4,10 @@
 # Daily Tests for ASP.NET Core SignalR
 # These use Sauce Labs resources, hence they run daily rather than per-commit.
 
+artifacts:
```

Comment:
This isn't in the right location. This needs to be under "jobs > parameters"

```diff
   parameters:
     buildDirectory: src/SignalR
     buildArgs: "/p:DailyTests=true /p:SauceUser='$(asplab-sauce-labs-username)' /p:SauceKey='$(asplab-sauce-labs-access-key)'"
     agentOs: Windows
     jobName: SignalRDailyTests
     jobDisplayName: "SignalR Daily Tests"
+    artifacts:
+      publish: true
+      path: 'src/SignalR/artifacts/'
```

Reply:
Yeah, I changed it on my internal testing branch and it ran but didn't look at that folder for artifacts